### ISSUE TITLE
[BUG] Color temperature not work with multiple controllers

### ISSUE THREAD

**Issue by AidenShaw2020** [xaviml/controllerx#316]
When I add another button, color temperature for this button not working, and I get warning in log.

* Devices involved:
* Model: [  IKEA 1810 ] as [ Light ]
* Integration: `z2m`
* AppDaemon version: v4.0.7
* ControllerX version: v4.8.0
* HACS version (if installed from there): Home Assistant OS 5.13
* Home Assistant Core version: core-2021.4.3

## AppDaemon app configuration
light_1_controller:
  module: controllerx
  class: E1810Controller
  controller: sensor.button_light1_action
  integration: z2m
  light: light.light_1
  actions:
    - arrow_left_hold
    - arrow_left_release
    - arrow_right_hold
    - arrow_right_release
    - arrow_right_click
    - arrow_left_click

light_2_controller:
  module: controllerx
  class: E1810Controller
  controller: sensor.button_light2_action
  integration: z2m
  light: light.light_2
  actions:
    - arrow_left_hold
    - arrow_left_release
    - arrow_right_hold
    - arrow_right_release
    - arrow_right_click
    - arrow_left_click
```

## Logs
2021-06-06 22:24:08.966341 INFO light_2_controller: 🎮 Button event triggered: `arrow_right_click`
2021-06-06 22:24:08.648108 WARNING light_2_controller: There is already an action executing for `arrow_right_click`. If you want a different behaviour change `mode` parameter, the default value is `single`.

**Comment by xaviml** [comment]
Hi @AidenShaw2020,

I'm sorry you're experiencing this problem. I recently learnt that this warning is shown when another error has been raised recently. However, that is another problem not related to this.

Could you please do the following:
- Download ControllerX v4.12.0. There should not be any breaking changes.
- Restart AppDaemon addon/server.
- Press arrow right and check logs.

If the error persist, can you share again the whole AppDaemon logs since you restarted? This way I will get the whole picture of the problem.

Regards,
Xavi M.

**Comment by AidenShaw2020** [comment]
Hi Xavi, 
I follow your advice, and update ControllerX to latest version, and now is all working OK.
Thank you for you help.

Have nice day, Aiden.

**Comment by AidenShaw2020** [comment]
Hi Xavi, i have "new" problem :)

I get same issues that I posted before, but I get this on third controller (svetlo_pracovna_controller).
[AppDaemon.log](https://github.com/xaviml/controllerx/files/6629390/AppDaemon.log)
[Error.log](https://github.com/xaviml/controllerx/files/6629392/Error.log)

Can you check it please?
Thanks.

**Comment by xaviml** [comment]
Hi @AidenShaw2020,

As indicated in the logs:

```
Value for `color_temp` attribute could not be retrieved from `light.svetlo_pracovna`. Check the FAQ to know more about this error: https://xaviml.github.io/controllerx/faq
```

This problem is shown when ControllerX cannot read the `color_temp` attribute from the `light.svetlo_pracovna`. The questions for you are:
- Does this light support color_temp?
- Can you change the color temperature through Home Assistant?
- Could you please share the attributes of this entity (Developer Tools > States)?

The problem is highly likely from the light, but depending on the answers to my questions, I will suggest you one thing or another.

Regards,
Xavi M.

**Comment by bluehallu** [comment]
I ran into the same problem, in my case it was because I was using light integration with platform: group, rather than the group integration.

**Comment by xaviml** [comment]
Hi all,

I have released ControllerX [v4.14.0](https://github.com/xaviml/controllerx/releases/tag/v4.14.0). This release contains a change on how ControllerX handles group entities like Light Groups or others. You can read more about it [here](https://xaviml.github.io/controllerx/advanced/entity-groups). This means that this problem might have disappeared with the new version, however, let me know if you are still experimenting issues with it. If so, please attach in the comment the AppDaemon logs so I can see what are the problems.

I will leave this ticket as closed for the moment, but reopen it or open a new one if needed.

Regards,

**Comment by bluehallu** [comment]
Can confirm after this update I went back to light groups and all is working as expected
